data/reusables/pull_requests/re-request-review.md

```diff
@@ -0,0 +1,2 @@
+You can re-request a review after you've made substantial changes to your pull request.
+Navigate to your pull request, in the right sidebar click on the {% octicon "sync" aria-label="The sync icon" %} icon to re-request a review.
```

Comment:
It might be worth borrowing the sentence structure from the topic on [pull requests](https://docs-953--docs897.herokuapp.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews#resolving-conversations). This also makes it easy to use some of the other terms that you suggested in the issue for better searchability, and makes sure that people know which tab of the pull request we're talking about.

```suggestion
To request a fresh review from a reviewer, in the sidebar of the **Conversation** tab, click the {% octicon "sync" aria-label="The sync icon" %} icon.
```